src/admin/python/lsst/qserv/admin/qservCli/launch.py

```diff
@@ -1166,22 +1168,26 @@ def entrypoint_help(
         The commands to get help for.
     qserv_image : `str`
         The name of the image to run.
+    entrypoint : `bool`
+        Show the entrypoint help.
+    spawned : `bool`
+        Show the spawned app help.
     dry : `bool`
         If True do not run the command; print what would have been run.
     """
-    args = [
-        "docker",
-        "run",
-        "--init",
-        "--rm",
-        qserv_image,
-        "entrypoint",
-    ]
-    if command:
-        args.append(command)
-    args.append("--help")
-    if dry:
-        print(" ".join(args))
-    else:
-        _log.debug('Running "%s"', " ".join(args))
-        subprocess.run(args, check=True)
+    if entrypoint:
+        print(f"Help output for 'entrypoint':\n")
```

Comment:
Does this need to be the `f`-string?
I also have an impression this printout is not neederd.

Reply:
it doesn't. thx.